receiver/prometheusexecreceiver/testdata/test_crasher.go

```diff
@@ -19,7 +19,7 @@ import (
 	"time"
 )
```

Comment:
I missed this before - shouldn't this file be in the `subprocessmanager/testdata` directory?